components/Blog/blogList.js

```diff
@@ -1,18 +1,17 @@
 import React from 'react'
-import './blog.scss'
 
 const BlogComponent = ({ content }) =>
-  (<div className='blog-component' >
+  (<div className='blog-component pa4' >
     <img src={content.imgUrl} />
     <div className='main-content'>
-      <div className='title'>{content.title}</div>
-      <div className='content'>{content.content}</div>
+      <div className='title tl pa3 fw6 ttc'>{content.title}</div>
```

Comment:
This should ideally be an h2 element for semantics (and the page should have an h1 that describes the page). If the title should be clickable, it should also be using a link to make it semantic and keyboard accessible.